nexus/sites.py

```diff
@@ -199,16 +203,15 @@ def media(self, request, module, path):
         # Respect the If-Modified-Since header.
         statobj = os.stat(fullpath)
         mimetype = mimetypes.guess_type(fullpath)[0] or 'application/octet-stream'
-        if not was_modified_since(request.META.get('HTTP_IF_MODIFIED_SINCE'),
-                                  statobj[stat.ST_MTIME], statobj[stat.ST_SIZE]):
+        if not was_modified_since(request.META.get('HTTP_IF_MODIFIED_SINCE'), statobj[stat.ST_MTIME]):
             return HttpResponseNotModified(content_type=mimetype)
         contents = open(fullpath, 'rb').read()
         response = HttpResponse(contents, content_type=mimetype)
         response["Last-Modified"] = http_date(statobj[stat.ST_MTIME])
         response["Content-Length"] = len(contents)
         return response
 
-    @never_cache
+    @method_decorator(never_cache)
```

Comment:
This is interesting. Is this required due to it being a class method? Why wasn't this required before?

Reply:
Yes, in 4.2 the test fails with this error: 
```
TypeError: never_cache didn't receive an HttpRequest. If you are decorating a classmethod, be sure to use @method_decorator.
```

Reply:
Thanks for the context. I wonder why that didn't fail before since this wasn't in the deprecation timeline. I wonder if this actually has to do with our type checker. Regardless, it looks like it's the right move